docs/source/data/errors.mdx

```diff
@@ -469,31 +488,32 @@ If it is necessary to change an error prior to reporting it to Apollo Studio (fo
 Consider an example where the error contains a piece of information like an API key:
 
 ```js
-throw new ApolloError("The x-api-key:12345 doesn't have sufficient privileges.");
+import { GraphQLError } from 'graphql';
+
+throw new GraphQLError(
+  "The x-api-key:12345 doesn't have sufficient privileges.",
+);
 ```
 
 The `rewriteError` function can ensure that such information is not sent to Apollo Studio and potentially revealed outside its intended scope:
```

Comment:
(this is now the default, ish)

Reply:
Tweaked the introduction to reflect this